sirepo/server.py

```diff
@@ -719,10 +725,27 @@ def _as_attachment(resp, content_type, filename):
 
 @pkconfig.parse_none
 def _cfg_db_dir(value):
-    """DEPRECATED"""
-    if value is not None:
-        srdb.server_init_root(value)
-    return srdb.root()
```

Comment:
@moellep, I am not sure about this change. Your guidance will be very helpful here. Thanks.

Reply:
@mrakitin, yes, I think it clobbered the srdb.py change - I'll fix this before merging.

Reply:
Thanks Paul! Feel free to push to An's PR branch.